ui/client/package.json

```diff
@@ -17,59 +17,61 @@
     "test": "./node_modules/karma/bin/karma start --single-run"
```

Comment:
Do you realy need update these all of packages? I think update shpuld be done in other PR.

Reply:
I have restored the old package.json and package-lock.json files. You should see a rebased version.